api-gateway/src/main/java/com/petclinic/bffapigateway/dtos/Visits/Status.java

```diff
@@ -0,0 +1,12 @@
+package com.petclinic.bffapigateway.dtos.Visits;
+
+public enum Status {
+
+    UPCOMING,
+    REQUESTED,
+    CONFIRMED,
+    CANCELLED,
+    IN_PROGRESS,
```

Comment:
Good stuff Fred, I. think we should maybe discuss removing one or two of the listed status option in here together for the next pull request